### DISCUSSION TITLE
Setting Object Page to Edit Mode

### DISCUSSION THREAD

**Discussion by qscheitlin** [UI5/webcomponents-react#4692] [Q&A] [answered]
Hello All, 

I have a quick question about the Object Page component.  When trying to use the object page in a create or edit scenario, what's the best (or intended) way to enable a user to set/define the 'headerTitle' prop.  Some of my initial ideas were relating to putting the Input component or TextArea component inside of the DynamicPageTitle, but that really doesn't seem like the intended design.

Thank you,
Quintin

**Comment by Lukas742** [comment, marked as answer]
Hi @qscheitlin 

I think this question is better placed in [SAP Fiori Community](https://answers.sap.com/questions/ask.html?primaryTagId=73554900100700000977), as we are not UX experts on Fiori design implementations and this use-case strikes me as very specific and I don't want to lead you in the wrong direction.

Please also note that according to [Fiori web design guidelines](https://experience.sap.com/fiori-design-web/object-page/) , it is not recommended to make the header editable:

> When you use the object page layout, you can also make the header editable when the page is in edit mode. However, try to avoid having an editable header and move the content from the header to the sections instead.

**Comment by qscheitlin** [comment]
Thanks so much for the quick response, @Lukas742.  The Fiori web design guidelines link you provided actually answered my question.  I'll keep the Fiori Community in mind for the future though, I was not aware of a dedicated community for Fiori. 

Thanks again!
